src/resources/locales/en.json

```diff
@@ -854,9 +854,229 @@
+                "NavigateNextOPDSPageAlt": {
+                    "description": "in library window for Catalogues (OPD) feeds, activates the next pagination links (alternative keyboard shortcut).",
+                    "name": "Go to Next OPDS Page (Alternative)"
+                },
+                "NavigateNextPage": {
+                    "description": "in reader window, move to previous or next \"page\" (or scroll unit, if pagination is disabled).",
+                    "name": "Go to Next Page"
+                },
+                "NavigateNextPageAlt": {
+                    "description": "in reader window, move to previous or next \"page\" (or scroll unit, if pagination is disabled).",
+                    "name": "Go to Next Page (Alternative)"
+                },
+                "NavigatePreviousChapter": {
+                    "description": "in reader window, move to previous \"chapter\" (spine item in the reading order of the publication). Note that the keyboard focus is automatically moved to the \"main\" area landmark, which is immediately before the frame that displays publication content (just like when following / activating a link in the table of contents).",
+                    "name": "Go to Previous Chapter"
+                },
+                "NavigatePreviousChapterAlt": {
+                    "description": "in reader window, move to previous \"chapter\" (spine item in the reading order of the publication). Note that the keyboard focus is automatically moved to the \"main\" area landmark, which is immediately before the frame that displays publication content (just like when following / activating a link in the table of contents).",
+                    "name": "Go to Previous Chapter (Alternative)"
+                },
+                "NavigatePreviousHistory": {
+                    "description": "in reader window, go back or forward in the navigation history: Beginning or End of the publication.",
+                    "name": "Go to Previous History"
+                },
+                "NavigatePreviousLibraryPage": {
+                    "description": "In the library view, display the previous set of books.",
+                    "name": "Go to Previous Library Page"
+                },
+                "NavigatePreviousLibraryPageAlt": {
+                    "description": "In the library view, display the previous set of books (alternative keyboard shortcut).",
+                    "name": "Go to Previous Library Page (Alternative)"
+                },
+                "NavigatePreviousOPDSPage": {
+                    "description": "in library window for Catalogues (OPD) feeds, activates the previous pagination links.",
+                    "name": "Go to Previous OPDS Page"
+                },
+                "NavigatePreviousOPDSPageAlt": {
+                    "description": "in library window for Catalogues (OPD) feeds, activates the previous pagination links (alternative keyboard shortcut).",
```

Comment:
typo: `opd`

Reply:
same as library page navigation, It is a previous/next history and a go to first/last history link for the alternative version